docs/source/new_project.rst

```diff
@@ -11,12 +11,26 @@ Choose where you would like to create your new project, and call copier with the
 
     >> copier gh:lincc-frameworks/python-project-template <path/to/destination>
 
+Copier will ask you questions for how to set up the project. These questions will be used to fill in aspects of the project's configuration, including both metadata and parameters. Below we provide some high-level overview of the questions:
+
+  * *What is the name of your project?* (``project_name``): The name of your project.
+  * *What is your python module name?* (``module_name``): The name of your (first) module. The main thing this controls is where your source code will live (``src/{{module_name}}``).
+  * *Your first and last name?* (``author_name``): The name of code's author.  This will be used in the project and documentation metadata.
+  * *Your preferred email address?* (``author_email``): The contact email for the code's author. This will be used in the project and documentation metadata.
+  * *What tooling would you like to use to enforce code style?* (``preferred_linter``): A linter is a tool to automatically format for consistency (see :doc:`Linting <../practices/linting>`). We provide options for `black <https://black.readthedocs.io/en/stable/>`_, `pylint <https://pypi.org/project/pylint/>`_, or no linter. Choosing a linter will include it as a project dependency and include it in the pre commit hooks.
```

Comment:
We have a :doc: for pre-commit that might be nice to link to here. Not sure how well known pre-commit is in the astro community.

Reply:
Added.